elements/pfe-styles/src/pfe-base.scss

```diff
@@ -82,47 +82,38 @@ th {
 // }
 
 body {
-  font-family: pfe-var(font-family);
-  font-size:   var( --pf-c--FontSize, pfe-var(content--FontSize));
-  font-weight: pfe-var(font-weight--light);
-  line-height: pfe-var(line-height);
+  @extend %pfe-content-base;
+  /* This variable is intentionally left empty for light DOM within components */
+  font-size: var( --pf-c--FontSize, pfe-var(content--FontSize));
   color: pfe-var(text);
   text-align: left;
   margin: 0;
 }
 
 // HTML TYPOGRAPHY
 .pfe-c-content {
-
   // HEADINGS
-  @each $heading-size in (h1, h2, h3, h4, h5, h6) {
-    #{$heading-size}  {
-        @extend %#{$heading-size}; 
+  @each $element in (h1, h2, h3, h4, h5, h6, p) {
+    #{$element}  {
+        @include pfe-content (#{$element}); 
     }
   }
 
   // TEXT & LISTS
   // paragraphs, list items, etc inherit from the body
-  p:not(:last-child):not(:empty) {
-      @extend %pfe-content--text--spacing-block;
-  }
-  small {
-      font-size: var( --pf-c--FontSize, pfe-var(content--small--FontSize));
-  } 
   dl,
   ol,
   ul {
-    @extend %list;
+    @extend %pfe-content--list;
   }
   dd,
   dt,
   li{
-      @extend %list-item;
+      @extend %pfe-content--list-item;
   }
   li >  ol,
   li >  ul {
-      @extend %nested-list-item;
-
+      @extend %pfe-content--nested-list-item;
```

Comment:
Oh I see my confusion - I think this needs to be named `%pfe-content--nested-list;` because above you used `list-item` to refer to the li tags and `list` to refer to the ul/ol tags.